source/geod24/LocalRest.d

```diff
@@ -444,6 +444,9 @@ private final class LocalScheduler : BaseFiberScheduler
 /// However, the one in `std.concurrency` is process-global (`__gshared`)
 private LocalScheduler scheduler;
 
+/// The main thread scheduler, used to wait for specific events
+private LocalScheduler main_scheduler;
```

Comment:
Why introduce another scheduler and not use `scheduler` instead ?

Reply:
This was just used to "mark" the thread as being the main thread. We can use a boolean instead. I'm pushing a small change.